src/Microsoft.AspNet.Mvc.TagHelpers/LinkTagHelper.cs

```diff
@@ -136,8 +138,8 @@ private enum Mode
         /// <remarks>
         /// If <c>true</c> then a query string "v" with the encoded content of the file is added.
         /// </remarks>
-        [HtmlAttributeName(FileVersionAttributeName)]
-        public bool? FileVersion { get; set; }
+        [HtmlAttributeName(AppendVersionAttributeName)]
+        public bool? AppendVersion { get; set; }
```

Comment:
I need a history lesson on this: why is this a nullable bool? Why not just a bool? Does it ever matter whether it was "not set" or "set to false"? I ask because in WebForms no controls ever needed this behavior (ok fine, and nullable didn't exist in .NET 1.0 either, but it was still possible to implement manually).